addons/stock_landed_costs/models/purchase.py

```diff
@@ -7,10 +7,3 @@ def _prepare_account_move_line(self, move=False):
         res = super()._prepare_account_move_line(move)
         res.update({'is_landed_costs_line': self.product_id.landed_cost_ok})
         return res
-
-    def _get_po_line_invoice_lines_su(self):
```

Comment:
I suggest we avoid unstable changes, even if the method has been very recently introduced
Just add a comment line like
```
#TODO master delete
```